src/main/java/icbm/classic/content/blocks/emptower/TileEMPTower.java

```diff
@@ -122,7 +123,7 @@ public boolean read(ByteBuf data, int id, EntityPlayer player, IPacket type) //T
                 }
                 case CHANGE_MODE_PACKET_ID:
                 {
-                    empMode = EMPMode.values()[data.readByte()];
```

Comment:
Its written as a byte in the description packet... this will cause it to read over the packet size

Reply:
It actually gets written as an integer